test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven/after/src/main/java/foo/bar/S3.java

```diff
@@ -151,8 +150,6 @@ private void singleBucketArgMethods(S3Client s3, String bucket) {
             .build());
         s3.deleteBucketCors(DeleteBucketCorsRequest.builder().bucket(bucket)
             .build());
-        s3.getBucketVersioning(GetBucketVersioningRequest.builder()
```

Comment:
Removed this because `bucket` is missing. Will create backlog item for this

Reply:
oops good catch, missed adding this in S3MethodsConstructorToFluent recipe. pushing fix to this branch